index.html

```diff
@@ -12,11 +12,11 @@
     </script>
     <script src="https://ucarecdn.com/widget/2.5.5/uploadcare/uploadcare.full.min.js"></script>
     <script src="https://cdn.rawgit.com/zenorocha/clipboard.js/v1.5.12/dist/clipboard.min.js"></script>
-    <script src="vendor/aframe-master.min.js"></script>
+    <script src="https://aframe.io/releases/1.3.0/aframe.min.js"></script>
     <script src="vendor/aframe-input-mapping-component.js"></script>
     <script src="build.js"></script>
     <script src="https://unpkg.com/aframe-gltf-exporter-component/dist/aframe-gltf-exporter-component.min.js"></script>
-    <script src="vendor/aframe-teleport-controls.min.js"></script>
+    <script src="vendor/aframe-blink-controls.min.js"></script>
```

Comment:
Maybe we can also move to the CDN https://cdn.jsdelivr.net/npm/aframe-blink-controls@0.3.0/dist/aframe-blink-controls.min.js